### DISCUSSION TITLE
Why does the graph V rotate？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#159] [Q&A] [answered]
In paper “Unsupervised random forests”：
![image](https://user-images.githubusercontent.com/5112397/148869910-6c459426-3b93-4ecb-aff5-2c1ef14a696e.png)
Y is obtained  by shifte x，Why does the graph V rotate in FIGURE 1  ？
![image](https://user-images.githubusercontent.com/5112397/148869639-adca4f72-eed6-403d-b2d0-b2e8c3a50f86.png)

**Comment by ishwaran** [comment, marked as answer]
The Y-values displayed have been transformed as part of the _sidification_ of the data.  This is described in detail in the paper.

**Comment by A-Pai** [comment]
您的邮件我已收到  I have received your email                        李刚  Li Gang
